### ISSUE TITLE
Add Accessibility tests and fix violations

### ISSUE THREAD

**Issue by sneridagh** [kitconcept/volto-light-theme#301]
@iRohitSingh I added #300 for the infrastructure, please add the basic tests, included the example content.
